### PR TITLE
RTR TowerBag

### PR DESCRIPTION
**Pull request by Ganimoth**
RTR Towerbag. Last one, I swear!

### CODE REVIEW

File: ports/towerbag/towerbag/Permission.png
Comment (JanTrueno) [inline ports/towerbag/towerbag/Permission.png:1]: Licenses are usually names like this: LICENSE.matheuscunegato.png

**Comment by cdeletre** [comment]
Some files are missing no? I don't see the gmloader and the libs 🙃

**Comment by Ganimoth** [comment]
> Some files are missing no? I don't see the gmloader and the libs 🙃

Those are already in existing release, I was under impression I am to only submit changed / new files? Do I get it wrong?

**Comment by cdeletre** [comment]
Oh my bad, I have missed that ! Sorry 😅

**Review by JanTrueno — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**